operator/Makefile

```diff
@@ -104,19 +104,19 @@ IMAGE_VERSION=${IMAGE_VERSION} \
 NAMESPACE="${NAMESPACE}" \
 VERBOSE_MODE="${VERBOSE_MODE}" \
 SERVICE_TYPE="${SERVICE_TYPE}" \
-envsubst | ${OC} apply -n ${OPERATOR_NAMESPACE} -f -
+envsubst | ${OC} apply -n "${OPERATOR_NAMESPACE}" -f -
 
 ## kiali-delete: Remove a Kiali CR from the cluster, informing the Kiali operator to uninstall Kiali.
 kiali-delete: secret-delete
 	@echo Remove Kiali
-	${OC} delete --ignore-not-found=true kiali kiali -n ${OPERATOR_NAMESPACE}
+	${OC} delete --ignore-not-found=true kiali kiali -n "${OPERATOR_NAMESPACE}"
 
 ## purge-kiali: Purges all Kiali resources directly without going through the operator or ansible.
 purge-kiali:
 	@echo Purge Kiali resources
-	${OC} patch kiali kiali -n ${OPERATOR_NAMESPACE} -p '{"metadata":{"finalizers": []}}' --type=merge ; true
-	${OC} delete --ignore-not-found=true all,secrets,sa,templates,configmaps,deployments,clusterroles,clusterrolebindings,ingresses,customresourcedefinitions --selector=app=kiali -n ${NAMESPACE}
-	${OC} delete --ignore-not-found=true oauthclients.oauth.openshift.io --selector=app=kiali -n ${NAMESPACE} ; true
+	${OC} patch kiali kiali -n "${OPERATOR_NAMESPACE}" -p '{"metadata":{"finalizers": []}}' --type=merge ; true
+	${OC} delete --ignore-not-found=true all,secrets,sa,templates,configmaps,deployments,clusterroles,clusterrolebindings,ingresses,customresourcedefinitions --selector=app=kiali -n "${NAMESPACE}"
+	${OC} delete --ignore-not-found=true oauthclients.oauth.openshift.io --selector=app=kiali -n "${NAMESPACE}" ; true
```

Comment:
why isn't selector option in these two commands also quoted? Seems like you needed to quote them elsewhere, but they are not here.. would it be best/consistent to say --selector="app=kiali" in these two lines too?

Reply:
Yes.